resources/diff-npm-package.ts

```diff
@@ -33,6 +33,9 @@ if (diff === '') {
   console.log('No changes found!');
 } else {
   const reportPath = localRepoPath('reports', 'npm-dist-diff.html');
+  if (!fs.existsSync) {
```

Comment:
@yaacovCR Did you test it?
Are you checking that `fs.existsSync` exist as a function or you checking if `reportPath` exists?

Also PR doesn't contain any description.

Reply:
Whoops! Sorry about that. Feel free to take over and fix as I won’t be able to address until several hours from now.